models/rate_neuron_ipn_impl.h

```diff
@@ -444,13 +444,13 @@ nest::rate_neuron_ipn< TNonlinearities >::handle(
       if ( e.get_weight() >= 0.0 )
       {
         B_.delayed_rates_ex_.add_value(
-          e.get_delay() - kernel().connection_manager.get_min_delay() + i,
+          e.get_delay() + i,
           e.get_weight() * e.get_coeffvalue( it ) );
       }
       else
       {
         B_.delayed_rates_in_.add_value(
-          e.get_delay() - kernel().connection_manager.get_min_delay() + i,
+          e.get_delay() + i,
```

Comment:
Could you give me a hint, why this is correct and does not break things? Have there also been some changes to the buffers? Because otherwise I would assume that changing the position where the values are stored in the buffers without changing the `get_value` counterpart in `update_` should result in wrong delays. (This obviously does not happen, but I am trying to understand why.)

Reply:
in the `update_` loop in `simulation_manager.cpp`, the timing of delivery has changed. while in 4g, events were gathered in one time step and only delivered in the next time step (hence the `- kernel().connection_manager.get_min_delay()`) in 5g, these two happen in the same time step. this means we do not have to substract the `min_delay` from the actual delay.